sdk/servicebus/Azure.Messaging.ServiceBus/tests/Infrastructure/ServiceBusScope.cs

```diff
@@ -122,7 +122,8 @@ public static async Task DeleteNamespaceAsync(string namespaceName)
                                                              bool enableSession,
                                                              bool forceQueueCreation = false,
                                                              TimeSpan? lockDuration = default,
-                                                             [CallerMemberName] string caller = "")
+                                                             [CallerMemberName] string caller = "",
+                                                             string overrideNamespace = default)
```

Comment:
You probably want to move this before `caller`, as that is never intended to be passed directly.